src/commands/help.js

```diff
@@ -0,0 +1,7 @@
+export default function(program) {
+  program
+    .command('help')
```

Comment:
Only thing I've noticed is that help gets appended to the list of available commands - unlike `slate version`. Anyone else bothered by this:

![image](https://cloud.githubusercontent.com/assets/991693/26155560/9909ee5a-3ae1-11e7-93d3-d82f9ac3a28f.png)

I'm wondering if this should be integrated as part of `@shopify/commander` instead of Slate CLI.

As of right now, you can't do `slate theme help` or `slate theme h`. Also, do you find `help` is even necessary or `-h` and `--help` is enough? Because right now we have 4 ways of getting help 😅.